api/src/test/java/org/openmrs/module/fhir2/api/search/AllergyIntoleranceSearchQueryImplTest.java

```diff
@@ -760,4 +764,91 @@ public void searchForAllergies_shouldHandleComplexQuery() {
 		    equalTo(PATIENT_UUID));
 	}
 	
+	@Test
+	public void shouldReturnCollectionOfAllergiesSortedBySeveritySevere() {
+		
+		SortSpec sort = new SortSpec();
+		sort.setParamName(AllergyIntolerance.SP_SEVERITY);
+		sort.setOrder(SortOrderEnum.ASC);
+		
+		IBundleProvider results = search(
+		 new SearchParameterMap().addParameter(FhirConstants.SEVERITY_SEARCH_HANDLER, FhirConstants.GLOBAL_PROPERTY_SEVERE, new TokenAndListParam().addAnd(new TokenParam(SEVERITY_SEVERE))).setSortSpec(sort));
```

Comment:
We don't need to add `FhirConstants.GLOBAL_PROPERTY_SEVERE`. Check the dao implementation.
Please use this for all other sorting tests added.